packages/graphql-hooks/src/GraphQLClient.ts

```diff
@@ -1,18 +1,108 @@
 import EventEmitter from 'events'
 import { extractFiles } from 'extract-files'
+import { Client } from 'graphql-ws'
```

Comment:
we should probably rename this to something more telling

Reply:
It was confusing for me to know what they are, especially since one is called `subscriptionCient` and this one `client` but the `subscriptionClient` one is the one that doesn't have a `subscribe` function (https://github.com/nearform/graphql-hooks/pull/799/files#diff-fa55352c78e085b512c50866f2aca7f24f34206b00aeee49f6e67250049403fbL345)